change/@fluentui-react-motion-798078cf-4718-44f2-8477-3b5d8540ee6b.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33091&commitId=53693a83ef6959d999d1812b3eb74229cfa86a31&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Basic High Contrast.chromium.png|2| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|13| Changed|
|Avatar Converged.badgeMask.chromium.png|5| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|4926| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>